modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -125,7 +123,7 @@ func (k Keeper) storeWasmCode(ctx sdk.Context, code []byte) ([]byte, error) {
 	}
 
 	// store the code hash
-	err = types.AddCodeHash(ctx, k.cdc, codeHash)
+	err = ibcwasm.CodeHashes.Set(ctx, codeHash)
 	if err != nil {
 		return nil, errorsmod.Wrap(err, "failed to store contract")
```

Comment:
not sure if this should be `"failed to set code hash"`, as an above wrapped error for `StoreCode` has the same `"failed to store contract"` string

Reply:
done